helpers/isWebThreeInstalled.ts

```diff
@@ -0,0 +1,13 @@
+declare global {
+  interface Window {
+    ethereum: any;
+  }
+}
+
+export default function isWebThreeInstalled() {
```

Comment:
you can summarize this into
```ts
export default function isWebThreeInstalled(): boolean { return !!window?.ethereum }

Reply:
I can't. Since `window?` equals to `window && window` and besides it came bool, could be `undefined` too, throwing the error in this case (bc it's using an undefined variable for verification).

Reply:
so: `window` will NEVER be udefined. No need to check for that, the null coallescent was there just so you could translate what I wrote for what you had. also, this function _has_ to return a boolean, you said so yourself here: https://github.com/bepronetwork/web-network/pull/9/files#diff-dd2381d8e5f35d95030468f5399bf2e507165edb616304f26aff894ccd5b013aR9

and while `null` is falsy, it's not a boolean. null is null. but since `window` will never be undefined, `return null` will never be read - so you dont need that there.

And since we know window will never be undefined, you also don't need the `if()`

leaving the fuction with `!!window.etherium` as its true form

Reply:
I'm sorry, I should have shown the recipe before:

![image](https://user-images.githubusercontent.com/16447765/124787345-cf9b3380-df1e-11eb-9e27-9116260a02f7.png)

Again, the summarized suggestion you provided do not work in this case. I return `null` for where we're using this modules do not receive `undefined` as well, when jump of through the `window` check,